src/relayEnv.js

```diff
@@ -13,7 +14,7 @@ function fetchQuery(
   // cacheConfig,
   // uploadables,
 ) {
-  return fetch('http://localhost:4000/graphql', {
```

Comment:
todo - do `API_URL | http://localhost:4000/graphql` as to not break build for everyone else